apps/frontend/src/components/LoginForm/LoginForm.module.css

```diff
@@ -1,57 +1,28 @@
-.formContainer {
-  margin: 0 auto;
-  width: 20vw;
-  min-height: 65vh;
-  min-width: min-content;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-evenly;
-  border: 1px black;
-  border-radius: 15px;
-  box-shadow: 15px 12px 12px 0px rgba(0,0,0,0.1);
-  background: white;
-  padding: 15px;
+h1 {
+  position: absolute;
+  top: 2rem;
+  right: 2rem;
```

Comment:
CSS modules are meant to be used only with classes.
You can notice that you don't have any `<h1>` in the LoginForm component. You have one in the LoginPage component.
Now, even though this `<h1>` styling was declared in the LoginForm CSS module, it applies its styling to any `<h1>` in our app.
Use a class name and apply it like that. It might be a bit tedious, but it's the only way we can encapsulate styling.